File: sem_5/flutter/task_manager/lib/models/Task.dart
class Task {
  String id;
  String title;
  String description;

  Task({required this.id, required this.title, required this.description});

  Map<String, dynamic> toJson() => {
        'id': id,
        'title': title,
        'description': description,
      };

  factory Task.fromJson(Map<String, dynamic> json) => Task(
        id: json['id'],
        title: json['title'],
        description: json['description'],
      );
}

File: sem_5/ueLibre/go/src/Model/Task.go
package Model

import "time"


type Task struct {
	ID 			uint 		`json:"id" gorm:"primaryKey"`
	Title 		string		`json:"title" gorm:"not null"`
	Description string		`json:"description"`
	Status		string		`json:"status" gorm:"default:'en cours'"`
	CreateAt	time.Time	`json:"created_at"`
	UpdateAt	time.Time	`json:"updated_at"`
}